packages/web/src/components/basic/URLParamsProvider.js

```diff
@@ -201,6 +201,9 @@ class URLParamsProvider extends Component {
 			// TODO: support for NestedList
 			if (value.location) return value;
 			if (value.category) return value;
+			if (value.mainLabel && value.secondaryLabel) {
```

Comment:
Only check the presence of `mainLabel` and `secondaryLabel` keys in value, if `true` then return `value` as it.